### DISCUSSION TITLE
How to set user profile via Postman API for the newly created users?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#330] [General Discussion]
I have created a tenant on one of our Sunbird projects.
The on boarding is done by setting up the taxonomy through APIs.
I have created users on Sunbird for the tenant through APIs.
I want to setup their profiles by selecting the preferences for BMGS through API without using their individual login.
Usually profile is setup from the frontend, but I want to set profile for a lot of users so I am looking for an API that can do this with Admin access.
I want to know is there an API that can achieve this ? Or any other way in which this can be done?
Please guide me here.

to: @reshmi-nair @rajeevsathish @Krishnaj20
cc: @baljeetsingh @komalm

**Comment by rajeevsathish** [comment]
@nidhikachhawa24  as we do not have any bulk api to change this as it is a individual user update by using user/v3/update of the user service.
@reshmi-nair can you please help on this if you have any other api in which you can do this task
cc: @Krishnaj20 @rhwarrier

**Reply by reshmi-nair** [reply]
@nidhikachhawa24 We don't have an API to update BGMS in bulk. But we can update name, phone, email, userType, roles, orgExternalId in bulk by creating a csv file and uploading it using /v1/user/upload API.
